web-components/src/index.ts

```diff
@@ -31,6 +31,7 @@ export { DatePickerDay } from "./components/datepicker/datepicker-day/DatePicker
 export { DatePickerMonth } from "./components/datepicker/datepicker-month/DatePickerMonth";
 export { DatePickerWeek } from "./components/datepicker/datepicker-week/DatePickerWeek";
 export { DateTimePicker } from "./components/date-time-picker/DateTimePicker";
+export { Dropdown } from "./components/dropdown/Dropdown";
```

Comment:
also add an export endpoint to webpack. Please read the CONTRIBUTING doc that Rost had put together

Reply:
Already done